posts/2025-02-17_theres_a_pharmaverse_package_for_that/managing-clinical-trial-data.qmd

```diff
@@ -0,0 +1,323 @@
+
+    -   [**admiral**](https://github.com/pharmaverse/admiral) for standardized data derivations.
+
+    -   [**metacore**](https://cran.r-project.org/web/packages/metacore/index.html) for metadata-driven structures.
+
+    -   [**dplyr**](https://dplyr.tidyverse.org/) (part of the tidyverse) for data manipulation
+
+4.  **Analysis Results** – Incorporating visualization and modeling tools:
+
+    -   [**ggplot2**](https://ggplot2.tidyverse.org/) (part of the tidyverse) for advanced data visualization.
+
+    -   [**visR**](https://github.com/openpharma/visR) for survival analysis and Kaplan-Meier plots.
+
+    -   [**pharmaRTF**](https://cran.r-project.org/web/packages/pharmaRTF/index.html) for creating submission-ready reports.
+
+    -   [**rtables**](https://github.com/insightsengineering/rtables) for structured table outputs
+
+    -   [**Tplyr**](https://github.com/atorus-research/Tplyr) for clinical summary tables
+
+    -   The pharmaverse offers multiple table-making packages, including [rtables](https://insightsengineering.github.io/rtables/v0.6.10/articles/rtables.html), [chevron](https://insightsengineering.github.io/chevron/latest-tag/), [pharmaRTF](https://atorus-research.github.io/pharmaRTF/), [gtsummary](https://www.danieldsjoberg.com/gtsummary/), [cards](https://insightsengineering.github.io/cards/latest-tag/), [tfrmt](https://gsk-biostatistics.github.io/tfrmt/), [tidytlg](https://pharmaverse.github.io/tidytlg/main/), and [cardinal](https://pharmaverse.github.io/cardinal/). You can find more tools on the [TLGs page](https://pharmaverse.org/e2eclinical/tlg/). Each serves different needs, and it is up to the programmer to choose the best fit.
+
+5.  **Submission** – Final reporting and submission:
+
+    -   [**xportr**](https://github.com/atorus-research/xportr) for CDISC-compliant dataset export.
+
+    -   [**pkglite**](https://merck.github.io/pkglite/) for package management and tracking.
+
+By using these tools, organizations can optimize their data pipeline, ensuring clinical data is well-structured, validated, and ready for regulatory submission with ease.
+
+## **Where Do I Start? Understanding Tasks & Objectives**
+
+For those new to clinical reporting in R, understanding the foundational tasks and objectives is critical.
+
+#### **Metadata-Driven Clinical Reporting Approach**
+
+A metadata-driven method ensures clinical trial data is consistently structured and aligned with regulatory standards. The process typically follows this sequence:
+
+[**Metadata**](https://pharmaverse.org/e2eclinical/metadata/) **\> [OAK](https://github.com/pharmaverse/sdtm.oak) \> [Admiral](https://github.com/pharmaverse/admiral) \> Define.xml \> [TLGs](https://pharmaverse.org/e2eclinical/tlg/) (rtf/pdf) \> Submissions \> Shiny**
+
+This structured approach makes data preparation for regulatory submission more transparent and traceable.
+
+#### **Essential Resources & Packages**
```

Comment:
as mentioned above: i'd absorb key elements of this list into the above list.

Reply:
yes agree - i find it confusing having 2 separate lists of select pharmaverse packages here. i would have one streamlined list with just a few examples, and encouraging people to go to our site to see the full offerings